R/add_highlight.R

```diff
@@ -0,0 +1,136 @@
+#' @title Highlight a specific strata
+#'
+#' @description S3 method for highlighting a specific strata by lowering the opacity of all other strata.
+#'
+#' @author Tim Treis
+#'
+#' @param gg visR object
+#' @param ... other arguments passed on to the method 
+#' 
+#' @examples
+#' library(visR)
+#'
+#' adtte %>%
+#'   visR::estimate_KM("SEX") %>%
+#'   visR::visr() %>%
+#'   visR::add_CI(alpha = 0.4) %>%
+#'   visR::add_highlight("SEX=M", bg_alpha_multiplier = 0.2)
```

Comment:
bg_alpha_mulitplier: 
- too long argument name
- you said you were going to decrease opacity of other strata so mulitplier is confusing
- perhaps we can do something like "alpha = xx" for the strata you want to highlight and you reduce accordingly

Reply:
I think this is a bit of a tricky issue.

I think the route of specifying the foreground stratas' `alpha` and then automatically reducing the others would be too strong of an opinion because the user might want to adjust this, depending on preference and colourscheme.

While I am decreasing the `alpha` of the background strata, I have written the code with `add_CI` in mind. Since the CI already have an `alpha`, chose to go with a multiplier instead of subtraction. This way, both the line as well as the CI are made more transparent proportionally.

But I fully agree that the name is too long :D I just wanted the name to be self-explanatory and didn't come up with something better yet.